src/useradd.c

```diff
@@ -2470,6 +2472,8 @@ static void create_mail (void)
 			return;
 		}
 
+		free(file);
```

Comment:
The (fd < 0) return above also requires a free(file).

Reply:
Fixed.  Thanks!